app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/UploadStatus.scala

```diff
@@ -14,6 +14,8 @@ object UploadStatus {
   case object Queued extends UploadStatus("QUEUED")
   case object Processing extends UploadStatus("PROCESSING")
   case object Complete extends UploadStatus("COMPLETE")
+  case object Failed extends UploadStatus("FAILED")
+  case object Aborted extends UploadStatus("ABORTED")
```

Comment:
I was originally dropping copying the data into a different column, dropping the first, dropping the type, creating the type, copying back, and dropping the new one before I realized i could just alter the column's type